lib/themes/dosomething/paraneue_dosomething/js/components/carousel.js

```diff
@@ -0,0 +1,30 @@
+const React = require('react');
+
+import ContextSlide from './context_slide.js';
+
+const Carousel = React.createClass({
```

Comment:
@DFurnes didn't they just deprecate the last established method for component creation @_@ haha